main.go

```diff
@@ -135,7 +144,7 @@ func RunBackup(conf *types.Conf) {
 	repos = bitbucket.Get(conf)
 	Backup(repos, conf)
 
-	conf.HasValidCronSpec()
```

Comment:
I would actually leave it there because I think it is useful to know when the next run will be.

Reply:
I refactored this in 1d7e22e so that it reads a little clearer and the log output isn't merely a side effect of an extra validation.